okhttp-logging-interceptor/src/main/java/okhttp3/logging/HttpLoggingInterceptor.java

```diff
@@ -285,6 +314,12 @@ public Level getLevel() {
     return response;
   }
 
+  private void logHeader(Headers headers, int i) {
+    String value =
+        headersToRedact.contains(headers.name(i).toLowerCase()) ? "<redacted>" : headers.value(i);
```

Comment:
Warning from the Ghost of Code Review Yet-to-Come:

https://github.com/square/okhttp/pull/4252#discussion_r216175280